src/DynamoCore/Core/Threading/UpdateRenderPackageAsyncTask.cs

```diff
@@ -183,17 +187,59 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, ref List<str
                         }
                     }
 
+                    var plane = graphicItem as Plane;
+                    if (plane != null)
```

Comment:
Yeah, the look of planes is definitely up for discussion. I should add the plane normal back, at the very least. I could also put back the original curves defining the plane, but at a color lighter than the default curve color.